packages/sdk/src/shared/index.ts

```
@@ -0,0 +1,844 @@
+};
+
+export type ComponentRequestResponse = { data: V1Component; };
+
+/**
+ * Different ways in which customers can authorize requests to HTTP endpoints
+ */
+export const enum HTTPAuthType {
+  None = "none",
+  StaticBearer = "static_bearer_token",
+  OAuth = "oauth",
+}
```

Biome

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)